src/holo-crypto-wasm/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "holo-crypto-wasm"
+version = "0.1.0"
+authors = ["Willem Olding <willemolding@gmail.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook"]
+
+[dependencies]
+wasm-bindgen = "0.2"
+
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.1", optional = true }
+
+# `wee_alloc` is a tiny allocator for wasm that is only ~1K in code size
+# compared to the default allocator's ~10K. It is slower than the default
+# allocator, however.
+#
+# Unfortunately, `wee_alloc` requires nightly Rust when targeting wasm for now.
+wee_alloc = { version = "0.4.2", optional = true }
+
+holo-config-core = { git = "https://github.com/Holo-Host/holo-config", rev = "d70f1f9b8fd7ab76df739abd12b9019ebef3ac44" }
```

Comment:
This can change to reference develop branch as soon as Holo-Host/holo-config#11 is merged